refactor_me.py

```diff
@@ -2,25 +2,49 @@
 
 from collections import namedtuple, defaultdict
 from operator import itemgetter
+from sys import stdout
+import io
 
-Expense = namedtuple('Expense', ('type_', 'amount'))
+Expense = namedtuple('Expense', ('category', 'amount'))
 
 
 def sum_expenses(expenses, min_amount=0):
+    """Aggregates all Expense entries into a categorized list of sums
+
+    Args:
+        expenses (iter(Expense)): Iterable of Expenses to be summed
+        min_amount (int): Minimum value to match to be included with the result
+
+    Returns:
+        dict: Category-grouped results of summed values
+    """
     aggregated_expenses = defaultdict(int)
-    for expense in expenses:
-        amount = expense.amount
+    for category, amount in expenses:
         if amount >= min_amount:
-            aggregated_expenses[expense.type_] += amount
+            aggregated_expenses[category] += amount
     return aggregated_expenses
 
 
-def print_expenses(expenses):
-    for expense, amount in sorted(expenses.items(), key=itemgetter(1)):
-        print(expense, amount)
+def format_dict(input_dict, write_out=None):
```

Comment:
function name does not comply with what function does

Reply:
Could you provide a different example. The method formats the dictionary and either outputs the results to terminal or to a file-like object. Doesn't just print, although it can.